src/main/java/io/github/minecraftcursedlegacy/api/vanillachecker/VanillaChecker.java

```diff
@@ -0,0 +1,19 @@
+package io.github.minecraftcursedlegacy.api.vanillachecker;
+
+import java.util.HashMap;
+
+import io.github.minecraftcursedlegacy.impl.vanillachecker.VanillaCheckerImpl;
+import net.minecraft.entity.player.Player;
+
+public class VanillaChecker {
+    private VanillaChecker(){/*No Constructor*/}
+
+    /**
+     * Checks if a player is using a Vanilla Client
+     * @param player
+     * @return
```

Comment:
Either add descriptive content to your `@return` and `@param player` don't have it :wink: